release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -21,12 +21,7 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 
 {product-title} {product-version} is supported on {op-system-base-full} 7.9 or later, as well as on {op-system-first} 4.8.
 
-You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base-full} 7.9 or later for compute machines.
```

Comment:
I'd like to keep this line -- the control plane still has to remain RHCOS, but the compute can be RHEL7.9+